docs/spec/dataclasses.rst

```diff
@@ -365,6 +365,12 @@ positional and may use other names.
 * ``alias`` is an optional str parameter that provides an alternative
   name for the field. This alternative name is used in the synthesized
   ``__init__`` method.
+* ``converter`` is an optional parameter that specifies a single-argument
+  callable used to convert values when assigning to the associated
+  attribute (Including the assignment in ``__init__``). The parameter
+  type of this single argument provides the type of the synthesized
```

Comment:
"The parameter type of this single argument" is confusing. Recommend simplifying to "The the of this parameter".